config/internal/webhook/sa_webhook.yaml.tmpl

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{.WebhookName}}
```

Comment:
given all of our other internal manifests, particularly SAs, are using the `ds-pipelines-XYZ-{{.Name}}` prefix, perhaps for cleanliness we should also be doing that here

Reply:
I don't think it applies to this in particular since Webhook will be same across DSP K8s Native installations. Thus we cannot stick the webhook name with any specific DSPA